apps/condo/domains/user/components/UserOrganizationsList.tsx

```diff
@@ -112,7 +113,7 @@ export const UserOrganizationsList = ({ userOrganization, organizationEmployeesQ
     )
 
     const list = useMemo(() => {
-        return userOrganizations.slice()
+        return uniqBy(userOrganizations, employee => get(employee, 'organization')).slice()
```

Comment:
```suggestion
        return uniqBy(userOrganizations, employee => get(employee, 'organization.id')).slice()
```
Do you should to check the uniqueness of `organization.id`?